src/schema/schema.i18n/ru.json

```diff
@@ -1,20 +1,19 @@
 {
-    "Title is required": "",
-    "Title must be a string": "",
-    "Title must be longer than 1 symbol": "",
-    "Title must be longer than 3 symbol": "",
-    "Comments's description is required": "",
-    "Comments's description must be a string": "",
-    "Comments's description must be longer than 1 symbol": "",
-    "Title must be longer than 2 symbols": "",
-    "Title can be 50 symbols maximum": "",
-    "Goal's title is required": "",
-    "Goal's title must be a string": "",
-    "Goal's description must be longer than 10 symbols": "",
-    "Goal's description is required": "",
-    "Goal's description must be a string": "",
-    "Goal's project or team are required": "",
-    "Criteria weight is required": "",
-    "Criteria weight must be longer than 1 symbol": "",
-    "Choose a dependency": ""
+    "Title is required": "Требуется заголовок",
+    "Title must be a string": "Заголовок должен быть строкой",
+    "Title must be longer than {upTo} symbol": "Заголовок должен быть длиннее {upTo} символов",
+    "Comments's description is required": "Требуется описание комментариев",
+    "Comments's description must be a string": "Описание комментариев должно быть строкой",
+    "Comments's description must be longer than 1 symbol": "Описание комментариев должно быть длиннее 1 символа",
+    "Title must be longer than 2 symbols": "Заголовок должен быть длиннее 2 символов",
+    "Title can be 50 symbols maximum": "Заголовок может содержать не более 50 символов",
+    "Goal's title is required": "Требуется название цели",
+    "Goal's title must be a string": "Название цели должно быть строкой",
+    "Goal's description must be longer than 10 symbols": "Описание цели должно быть длиннее 10 символов",
+    "Goal's description is required": "Требуется описание цели",
+    "Goal's description must be a string": "Описание цели должно быть строкой",
+    "Goal's project or team are required": "Требуется проект или команда цели",
+    "Criteria weight is required": "Требуется вес критериев",
+    "Criteria weight must be longer than 1 symbol": "Вес критерия должен быть длиннее 1 символа",
```

Comment:
I think this key is incorrect. It's possible to have first criteria with 99 weight and second with 1 weight

cc @LamaEats

Reply:
I didn't understand🙃 
what should be written instead of this key?